ci/env/kava-internal-testnet/genesis.json

```diff
@@ -2022,32 +2022,7 @@
             "decimals": 6
           }
         ],
-        "enabled_conversion_pairs": [
-          {
-            "kava_erc20_address": "0xBb304f44b7EFD865361F2AD973d8ebA433893ABC",
-            "denom": "erc20/multichain/usdc"
-          },
-          {
-            "kava_erc20_address": "0xA637F4CECbA91Ad19075bA3d330cd95f694B1707",
-            "denom": "erc20/multichain/usdt"
-          },
-          {
-            "kava_erc20_address": "0x288429bc07B8d030ba418bb30F170327F9fBE502",
-            "denom": "erc20/multichain/wbtc"
-          },
-          {
-            "kava_erc20_address": "0x7a5DBf8e6ac1F6aCCF14f5B4E88b21EAA04c983d",
-            "denom": "erc20/axelar/usdc"
-          },
-          {
-            "kava_erc20_address": "0x7d2Ee2914324d5D4dC33A5c295E720659D5F3fA7",
-            "denom": "erc20/axelar/wbtc"
-          },
-          {
-            "kava_erc20_address": "0x5d6D67a665C9F169B0f9436E05B11108C1606043",
-            "denom": "erc20/axelar/eth"
-          }
-        ]
+        "enabled_conversion_pairs": []
```

Comment:
removed these to make it clear that they'll be overwritten in via a proposal in the seed script.